pyluba/aliyun/cloud_gateway.py

```diff
@@ -50,15 +50,19 @@ class CloudIOTGateway:
     _iotCredentials = {}
     converter = DatatypeConverter()
 
+    _regions_file_path = 'regions.json'
+    _iotCredentials_file_path = 'iotCredentials.json'
+    _mqtt_credentials_file_path = '_mqtt_credentials.json'
+
     def __init__(self):
         self._region = {}
         self._app_key = "34231230"
         self._app_secret = "1ba85698bb10e19c6437413b61ba3445"
         self.domain = 'api.link.aliyun.com'
 
         uuid1 = str(uuid.uuid1())  # 128 chatarrers
-        self._client_id = self.generate_random_string(8)  # First 8 Characters
-        self._device_sn = self.generate_random_string(32)  # 32 Characters
+        self._client_id = uuid1[:8]  # First 8 Characters
```

Comment:
Why has this been changed back? Generate random string matches the method in the sdk/app

Reply:
yes, in the app they are random if the securityguardmamanger function returns null (I think it is a null check, they should be hw based generated parameters). But when debugging the app they are always the same for each device. If we keep them the same, the triplet parameters (mqtt) do not change and we do not have to repeat the binding procedure every time (it seems to me that there is a maximum limit of devices in binding for each account, 10 if I remember correctly)

Reply:
random string or random uuid we need to keep it recorded then and saved as a one off operation, so instead check if a uuid exists on file and if not generate one to be re-used.

Reply:
or even better create a hardware fingerprint